### PR TITLE
XML comment clean up in MSALWrapper and MSALWrapperTest

### PR DESCRIPTION
**Pull request by melindajohnson**
Cleaning up the XML comments in MSALWrapper and removing XML comments in MSALWrapper Test.

### CODE REVIEW

File: src/MSALWrapper/AuthMode.cs [hunk 1]
```diff
@@ -5,64 +5,40 @@ namespace Microsoft.Authentication.MSALWrapper
...
-    /// The auth mode.
-    /// </summary>
+    /// <summary>/// The auth mode./// </summary>
     [Flags]
     public enum AuthMode : short
     {
-        /// <summary>
-        /// The web.
-        /// </summary>
+        /// <summary>/// The web./// </summary>
```
Comment (shalinikhare27) [inline src/MSALWrapper/AuthMode.cs:12]: We want the summary to be more meaningful than "The web". These were added by me using resharper which does a poor job of creating meaingful summaries but adds a placeholder.

File: src/MSALWrapper/ExceptionExtensions.cs [hunk 2]
```diff
@@ -41,15 +36,9 @@ public static string ToFormattedString(this Exception exception)
     /// <summary>
     /// The get all exceptions.
```
Comment (shalinikhare27) [inline src/MSALWrapper/ExceptionExtensions.cs:37]: Definitely needs an update. ;)

**Comment by shalinikhare27** [comment]
Sorry I didn't see this is in draft. I will wait for it to go live. The summaries can be improved to more meaningful content.

File: src/MSALWrapper/JsonWebTokenExtensions.cs [hunk 3]
```diff
@@ -9,12 +9,12 @@ namespace Microsoft.Authentication.MSALWrapper
     using Microsoft.IdentityModel.JsonWebTokens;
 
     /// <summary>
-    /// The json web token extensions.
+    /// json web token extensions.
```
Comment (goagain) [inline src/MSALWrapper/JsonWebTokenExtensions.cs:12]:
```suggestion
    /// Json web token extensions.

```

File: src/MSALWrapper/ITokenFetcher.cs [hunk 4]
```diff
@@ -15,61 +15,42 @@ public interface ITokenFetcher
...
         /// The get access token async.
         /// </summary>
-        /// <param name="authMode">
-        /// The auth mode.
-        /// </param>
-        /// <returns>
-        /// The <see cref="Task"/>.
-        /// </returns>
+        /// <param name="authMode">/// The auth mode.</param>
+        /// <returns>/// The <see cref="Task"/>.</returns>
```
Comment (goagain) [inline src/MSALWrapper/ITokenFetcher.cs:31]:
```suggestion
        /// <returns>The <see cref="Task"/>.</returns>
```

File: src/MSALWrapper/ITokenFetcher.cs [hunk 5]
```diff
@@ -15,61 +15,42 @@ public interface ITokenFetcher
...
-        /// The scopes.
-        /// </param>
-        /// <param name="authMode">
-        /// The auth mode.
-        /// </param>
-        /// <returns>
-        /// The <see cref="Task"/>.
+        /// <param name="scopes">/// The scopes.</param>
+        /// <param name="authMode">/// The auth mode.</param>
+        /// <returns>/// The <see cref="Task"/>.
```
Comment (goagain) [inline src/MSALWrapper/ITokenFetcher.cs:39]:
```suggestion
        /// <returns>
        /// The <see cref="Task"/>.
```

File: src/MSALWrapper/ITokenFetcher.cs [hunk 6]
```diff
@@ -15,61 +15,42 @@ public interface ITokenFetcher
...
-        /// <param name="scopes">
-        /// The scopes.
-        /// </param>
-        /// <param name="authMode">
-        /// The auth mode.
-        /// </param>
-        /// <returns>
-        /// The <see cref="Task"/>.
+        /// <param name="scopes">/// The scopes.</param>
+        /// <param name="authMode">/// The auth mode.</param>
```
Comment (goagain) [inline src/MSALWrapper/ITokenFetcher.cs:38]:
```suggestion
        /// <param name="scopes">The scopes.</param>
        /// <param name="authMode">The auth mode.</param>
```

File: src/MSALWrapper/ITokenFetcher.cs [hunk 7]
```diff
@@ -15,61 +15,42 @@ public interface ITokenFetcher
...
         /// <summary>
         /// The get access token async.
         /// </summary>
-        /// <param name="authMode">
-        /// The auth mode.
-        /// </param>
-        /// <returns>
-        /// The <see cref="Task"/>.
-        /// </returns>
+        /// <param name="authMode">/// The auth mode.</param>
```
Comment (goagain) [inline src/MSALWrapper/ITokenFetcher.cs:30]:
```suggestion
        /// <param name="authMode">The auth mode.</param>
```

File: src/MSALWrapper/ITokenFetcher.cs [hunk 8]
```diff
@@ -15,61 +15,42 @@ public interface ITokenFetcher
...
         /// The get access token async.
         /// </summary>
-        /// <param name="scopes">
-        /// The scopes.
-        /// </param>
-        /// <returns>
-        /// The <see cref="Task"/>.
-        /// </returns>
+        /// <param name="scopes">/// The scopes.</param>
+        /// <returns>/// The <see cref="Task"/>.</returns>
```
Comment (goagain) [inline src/MSALWrapper/ITokenFetcher.cs:47]:
```suggestion
        /// <param name="scopes">The scopes.</param>
        /// <returns>The <see cref="Task"/>.</returns>
```

File: src/MSALWrapper/ITokenFetcher.cs [hunk 9]
```diff
@@ -15,61 +15,42 @@ public interface ITokenFetcher
...
         Task<TokenResult> GetAccessTokenAsync(IEnumerable<string> scopes);
 
         /// <summary>
         /// Clear the local token cache for the current public client application.
         /// </summary>
-        /// <returns>
-        /// The <see cref="Task"/>.
-        /// </returns>
+        /// <returns>/// The <see cref="Task"/>.</returns>
```
Comment (goagain) [inline src/MSALWrapper/ITokenFetcher.cs:53]:
```suggestion
        /// <returns>The <see cref="Task"/>.</returns>
```

**Review by goagain — CHANGES_REQUESTED** [review]

File: src/MSALWrapper/AuthMode.cs [hunk 10]
```diff
@@ -73,28 +69,20 @@ public static bool IsBroker(this AuthMode authMode)
...
+        /// Checks if authMode is web.
         /// </summary>
-        /// <param name="authMode">
-        /// The auth mode.
-        /// </param>
-        /// <returns>
-        /// The <see cref="bool"/>.
-        /// </returns>
+        /// <param name="authMode">/// The auth mode.</param>
+        /// <returns>/// The <see cref="bool"/>.</returns>
```
Comment (goagain) [inline src/MSALWrapper/AuthMode.cs:75]:
```suggestion
        /// <param name="authMode">The auth mode.</param>
        /// <returns>The <see cref="bool"/>.</returns>
```

File: src/MSALWrapper/AuthMode.cs [hunk 11]
```diff
@@ -73,28 +69,20 @@ public static bool IsBroker(this AuthMode authMode)
...
+        /// Checks if authMode is device code.
         /// </summary>
-        /// <param name="authMode">
-        /// The auth mode.
-        /// </param>
-        /// <returns>
-        /// The <see cref="bool"/>.
-        /// </returns>
+        /// <param name="authMode">/// The auth mode.</param>
+        /// <returns>/// The <see cref="bool"/>.</returns>
```
Comment (goagain) [inline src/MSALWrapper/AuthMode.cs:85]:
```suggestion
        /// <param name="authMode">The auth mode.</param>
        /// <returns>The <see cref="bool"/>.</returns>
```

File: src/MSALWrapper/AuthenticationTimeoutException.cs [hunk 12]
```diff
@@ -6,16 +6,14 @@ namespace Microsoft.Authentication.MSALWrapper
...
     /// </summary>
     public class AuthenticationTimeoutException : Exception
     {
         /// <summary>
         /// Initializes a new instance of the <see cref="AuthenticationTimeoutException"/> class.
         /// </summary>
-        /// <param name="message">
-        /// The message.
-        /// </param>
+        /// <param name="message">/// The message.</param>
```
Comment (goagain) [inline src/MSALWrapper/AuthenticationTimeoutException.cs:16]:
```suggestion
        /// <param name="message">The message.</param>
```

File: src/MSALWrapper/ExceptionListToStringConverter.cs [hunk 13]
```diff
@@ -13,14 +13,10 @@ namespace Microsoft.Authentication.MSALWrapper
...
-        /// The execute.
+        /// Executes the convertion of exceptions to a string.
         /// </summary>
-        /// <param name="exceptions">
-        /// The exceptions.
-        /// </param>
-        /// <returns>
-        /// The <see cref="string"/>.
-        /// </returns>
+        /// <param name="exceptions">/// The exceptions.</param>
```
Comment (goagain) [inline src/MSALWrapper/ExceptionListToStringConverter.cs:18]:
```suggestion
        /// <param name="exceptions">The exceptions.</param>
```

**Review by goagain — CHANGES_REQUESTED** [review]
You can use `Add to batch` in tab `File Changed` to combine them into one commit.

**Review by kyle-rader — DISMISSED** [review]

**Review by mvanchaa — APPROVED** [review]

**Comment by reillysiemens** [comment]
To answer your question, @goagain, I tend to write these things as imperatives, so I'd write something like
```
Get a display name.
```
I agree that we should be consistent here.

**Review by reillysiemens — APPROVED** [review]

**Review by reillysiemens — APPROVED** [review]

**Review by mvanchaa — APPROVED** [review]

**Review by goagain — APPROVED** [review]

**Pull request merged**